terraform/cloudwatch_dashboard.template.json.py

```diff
@@ -802,29 +788,23 @@ def dashboard_body(name: str):
                             }
                         ],
                         [
-                            'AWS/Lambda',
-                            'Errors',
-                            'FunctionName',
+                            '...',
```

Comment:
Separate commit for this refactoring. I think it should be the first commit on this PR.